### ISSUE TITLE
NUmber of leads or entries appearing on the screen

### ISSUE THREAD

**Issue by patricechataigner** [oiime/sidr#26]
Currently we can only show 10, 25, 50 or 100 leads or entries at once.
Is it possible to have a more customizable way to show leads or entries? in https://github.com/huridocs/OpenEvSys for instance they allow  for different ways to see the leads, see file enclosed

[151123 number of leads or entries.docx](https://github.com/oiime/sidr/files/68244/151123.number.of.leads.or.entries.docx)

**Comment by oiime** [comment]
Yeah, wont be a problem to add. anything else to do on the way as I'm currently waiting for the test run to get another batch of issues?
